client/js/components/map/admin/DpMapAdminScales.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <div class="space-stack-xs">
+    <dp-multiselect
+      v-model="scales"
+      label="label"
+      multiple
+      :options="availableScales"
+      track-by="value"
+      @input="sortSelected('scales')">
```

Comment:
The sortSelected method seems to be missing, I get a lot of errors in the console

Reply:
thanks for trying this out... as always, it does not pay off to change things blindly. 33159f1b54139a79d90d4f28776a2ac8bbb33205

Reply:
:scream: